spec/system/user_flipped_to_not_approved_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+context 'when a user is logged in' do
+  include_context 'log in site admin'
+
+  context 'when the user is set to not_approved' do
+    before do
+      # confirm that the user is logged in
+      expect(page).to have_content('Add Site')
```

Comment:
Rubocop flags this line: "Do not use `expect` in `before` hook". I was intending that as a paranoid sanity-check, but as it's not really necessary, and it *is* a good idea not to set expectations in a `before` block, we should probably remove it.

Reply:
Removed